recipe-server/client/control_new/components/common/ShieldIdenticon.js

```diff
@@ -3,52 +3,51 @@ import PropTypes from 'prop-types';
 
 import Genome from 'control_new/utils/Genome';
 
-// Solarized colors
-export const colors = [
-  '#002b36', // base03
-  '#073642', // base02
-  '#586e75', // base01
-  '#657b83', // base00
-  '#839496', // base0
-  '#93a1a1', // base1
-  '#eee8d5', // base2
-  '#fdf6e3', // base3
-  '#b58900', // yellow
-  '#cb4b16', // orange
-  '#dc322f', // red
-  '#d33682', // magenta
-  '#6c71c4', // violet
-  '#268bd2', // blue
-  '#2aa198', // cyan
-  '#859900', // green
+
+const emojis = [
+  '😄', '😃', '😀', '😊', '😉', '😍', '😘', '😚', '😗', '😙', '😜', '😝', '😛', '😳', '😁', '😔',
+  '😌', '😒', '😞', '😣', '😢', '😂', '😭', '😪', '😥', '😰', '😅', '😓', '😨', '😱', '😠', '😡',
+  '😤', '😖', '😆', '😋', '😷', '😎', '😴', '😵', '😲', '😟', '😦', '😧', '😈', '👿', '😮', '😬',
+  '😐', '😯', '😶', '😇', '😏', '😑', '👼', '😺', '😻', '😽', '😼', '🙀', '😿', '😹', '😾', '👹',
+  '👺', '🙈', '🙉', '🙊', '💀', '👽', '💩', '🔥', '✨', '🌟', '💫', '💥', '💦', '💧', '💤', '👂',
+  '👀', '👃', '👅', '👄', '👍', '👎', '👌', '👊', '✊', '👋', '✋', '👐', '👆', '🙌', '🙏', '👏',
+  '💪', '💃', '🎩', '👑', '👒', '👟', '👞', '👡', '👠', '👢', '💼', '👜', '👝', '👛', '👓', '🎀',
+  '🌂', '💄', '💛', '💙', '💜', '💚', '💔', '💗', '💓', '💕', '💖', '💞', '💘', '💌', '💋', '💍',
+  '💎', '👣', '🐶', '🐺', '🐱', '🐭', '🐹', '🐰', '🐸', '🐯', '🐨', '🐻', '🐷', '🐽', '🐮', '🐗',
+  '🐵', '🐒', '🐴', '🐑', '🐘', '🐼', '🐧', '🐦', '🐤', '🐥', '🐣', '🐔', '🐍', '🐢', '🐛', '🐝',
+  '🐜', '🐞', '🐌', '🐙', '🐚', '🐠', '🐟', '🐬', '🐳', '🐋', '🐄', '🐏', '🐀', '🐃', '🐅', '🐇',
+  '🐉', '🐎', '🐐', '🐓', '🐕', '🐖', '🐁', '🐂', '🐲', '🐡', '🐊', '🐫', '🐪', '🐆', '🐈', '🐩',
+  '🐾', '💐', '🌸', '🌷', '🍀', '🌹', '🌻', '🌺', '🍁', '🍃', '🍂', '🌿', '🌾', '🍄', '🌵', '🌴',
+  '🌲', '🌳', '🌰', '🌱', '🌼', '🌐', '🌞', '🌝', '🌚', '🌜', '🌛', '🌙', '🌍', '🌎', '🌏', '⭐',
+  '⛅', '⛄', '🌀', '💝', '🎒', '🎓', '🎏', '🎃', '👻', '🎄', '🎁', '🎋', '🎉', '🎈', '🔮', '🎥',
+  '📷', '📹', '📼', '💿', '📀', '💽', '💾', '💻', '📱', '📞', '📟', '📠', '📡', '📺', '📻', '🔊',
+  '🔔', '📢', '⏳', '⏰', '🔓', '🔒', '🔏', '🔐', '🔑', '🔎', '💡', '🔦', '🔆', '🔅', '🔌', '🔋',
+  '🔍', '🛁', '🛀', '🚿', '🚽', '🔧', '🔨', '🚪', '💣', '🔫', '🔪', '💊', '💉', '💰', '💸',
+  '📨', '📬', '📌', '📎', '📕', '📓', '📚', '📖', '🔬', '🔭', '🎨', '🎬', '🎤', '🎵', '🎹', '🎻',
+  '🎺', '🎷', '🎸', '👾', '🎮', '🃏', '🎲', '🎯', '🏈', '🏀', '⚽', '🎾', '🎱', '🏉', '🎳', '⛳',
+  '🚴', '🏁', '🏇', '🏆', '🎿', '🏂', '🏄', '🎣', '🍵', '🍶', '🍼', '🍺', '🍻', '🍸', '🍹', '🍷',
+  '🍴', '🍕', '🍔', '🍟', '🍗', '🍤', '🍞', '🍩', '🍮', '🍦', '🍨', '🍧', '🎂', '🍰', '🍪', '🍫',
+  '🍬', '🍭', '🍯', '🍎', '🍏', '🍊', '🍋', '🍒', '🍇', '🍉', '🍓', '🍑', '🍌', '🍐', '🍍', '🍆',
+  '🍅', '🌽', '🏠', '🏡', '⛵', '🚤', '🚣', '🚀', '🚁', '🚂', '🚎', '🚌', '🚍', '🚙', '🚘', '🚗',
+  '🚕', '🚖', '🚛', '🚚', '🚨', '🚓', '🚔', '🚒', '🚑', '🚐', '🚲', '🚜', '💈', '🚦', '🚧', '🏮',
+  '🎰', '🗿', '🎪', '🎭', '📍', '🚩', '💯',
```

Comment:
A small trick, if we wanted this to be more compact and easier to edit:

```js
const emojis = `
  😄😃😀😊😉😍😘😚
  😗😙😜😝😛😳😁😔`
  .split(/\s*/));
```

On the other hand, we should probably avoid changing this much, since it would make the same seed generate different colors after the change. We probably don't want to do this. Overall we probably can't change the identishields much after we deploy them.